WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -120,38 +120,23 @@ protected Spanned doInBackground(Void... params) {
             String postTitle = "<h1>" + mPost.getTitle() + "</h1>";
             String postContent = postTitle + mPost.getContent();
 
-            if (mPost.isLocalDraft()) {
-                contentSpannable = WPHtml.fromHtml(
-                        postContent.replaceAll("\uFFFC", ""),
-                        getActivity(),
-                        mPost,
-                        Math.min(mTextView.getWidth(), mTextView.getHeight())
-                );
-            } else {
-                String htmlText = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?><html><head><link rel=\"stylesheet\" "
-                                  + "type=\"text/css\" href=\"webview.css\" /></head><body><div "
-                                  + "id=\"container\">%s</div></body></html>";
-                htmlText = String.format(htmlText, StringUtils.addPTags(postContent));
-                contentSpannable = new SpannableString(htmlText);
-            }
+            String htmlText = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?><html><head><link rel=\"stylesheet\" "
+                              + "type=\"text/css\" href=\"webview.css\" /></head><body><div "
+                              + "id=\"container\">%s</div></body></html>";
+            htmlText = String.format(htmlText, StringUtils.addPTags(postContent));
+            contentSpannable = new SpannableString(htmlText);
 
             return contentSpannable;
         }
 
         @Override
         protected void onPostExecute(Spanned spanned) {
             if (mPost != null && spanned != null) {
-                if (mPost.isLocalDraft()) {
-                    mTextView.setVisibility(View.VISIBLE);
-                    mWebView.setVisibility(View.GONE);
-                    mTextView.setText(spanned);
-                } else {
-                    mTextView.setVisibility(View.GONE);
-                    mWebView.setVisibility(View.VISIBLE);
-
-                    mWebView.loadDataWithBaseURL("file:///android_asset/", spanned.toString(),
-                                                 "text/html", "utf-8", null);
-                }
+                mTextView.setVisibility(View.GONE);
```

Comment:
I think we can remove `mTextView` now since we always end up hiding it. It might be a good opportunity to clean up this `Fragment` a bit. I ended up making a bunch of changes while I was trying things out, so I thought to just commit it somewhere to share some proposed changes with you: https://github.com/wordpress-mobile/WordPress-Android/commit/c52dfc0100d36276d67465113ad85b607c44461c

We don't have to make any of those changes of course, so it's totally up to you. Feel free to cherry-pick the commit or just re-commit in this branch if you do like it.

Reply:
great suggestions! I'll cherry-pick the commit and let you know 👍